cdc/entry/schema_storage_test.go

```diff
@@ -136,8 +136,8 @@ func (*schemaSuite) TestTable(c *check.C) {
 				Length: 10,
 			},
 		},
-		Unique:  true,
-		Primary: true,
+		Unique:  false,
+		Primary: false,
```

Comment:
This is a bug in original unit test, the test adds an index to an ineligible table in L195, the index was expected as non unique, which is tested in L223